### ISSUE TITLE
"Add breakpoint at current line" but no "Remove breakpoint at current line" 

### ISSUE THREAD

**Issue by jaykz52** [pdcgomes/XCActionBar#13]
Using the "Add breakpoint" command when current line is activated does remove the breakpoint, so possibly the action can just be renamed?

**Comment by pdcgomes** [comment]
Yeah, known issue - Menu Bar items aren't currently being updated when their titles changes in the menubar (enabled/disabled state does thought) - I've got the hooks in place but haven't gotten around to actually to wire and test things, I'll get to it soon enough, it's on my list, so stay tuned!

Most actions still work though since they use the same underlying target/action.

**Comment by jaykz52** [comment]
:+1: Is there a ticket to cover that? Or should I just leave this ticket open to represent that work?

**Comment by pdcgomes** [comment]
No, just tracking it on my internal todo list. Leave this one around and I'll be sure to reference it once I have this fixed.
